.github/tests/test_organization.py

```diff
@@ -75,42 +75,19 @@ def test_schema_naming(tmp_path):
             # Assert it loads with jsonschema
             schema_file = os.path.join(schema_dir, schema)
             loaded = load_schema(schema_file)
-
+            # print(json.dumps(loaded, indent=2))
```

Comment:
should this be commented out (or just deleted, better?)

Reply:
my bad, that was me troubleshooting. I got this fixed now

Reply:
Was this removed? I approved, but conditional on removing this line.

Reply:
This is fixed now